static/templates/message_controls.hbs

```diff
@@ -24,7 +24,7 @@
 
```

Comment:
I think the Cancel button in the `message_failed` thing above needs the role change as well.

Reply:
added